src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String toString() {
```

Comment:
I belive Override annotation should be defined above toString(). Also there are not requirement for custom toString, so please use default one (right click -> gererate -> toString)